### PR TITLE
Feature/functionality

### LINKED ISSUE #28: Complete the Functionality of Client Side App

### PR DESCRIPTION
**Pull request by holladayian**
## Description
This PR "completes" the functionality for the MVP from the client side. It conditionally renders a page, based on if a user isGen. It will also only allow a user to continue forward if the player count is greater than 1, and then if a word to guess is chosen.

## Where To Start
This touches all three main components, `App` `WordSelector` and `Gamepage`

## Affected areas of application
See above

## Screenshots (if applicable):
Needed state to view Gamepage for generator 
Screen Shot 2020-10-28 at 12 28 28 PM

Game page for Generator
Screen Shot 2020-10-28 at 12 29 14 PM

Needed state to view Gamepage for guesser
Screen Shot 2020-10-28 at 12 29 32 PM

Gamepage for Guesser
Screen Shot 2020-10-28 at 12 29 51 PM

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [x] DOM works correctly upon proper state

## Relevant Tickets
Fixes #28, #29, #30 

## Tags
@Chriscastanuela 

## Questions or Notes (if applicable):
n/a

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,38 +3,51 @@ import { Switch, Route } from 'react-router-dom';
...
     super()
     this.state = {
-      isGuesser: null,
+      isGenerator: null,
+      numPlayers: 0,
       word: '',
-      guess: ''
+      guess: '',
+      // attempts: [],
+      attempts: ['cuts', 'butts', 'coconuts'],
```
Comment (GreyMatteOr) [inline src/App.js:19]: rap god

File: src/Gamepage/Gamepage.js [hunk 2]
```diff
@@ -8,30 +8,73 @@ export class Gamepage extends Component {
...
+
+  mapAttempts = () => {
+    return this.props.attempts.map(attempt => {
+      return <p>{attempt}</p>
+    })
+  }
+
+  splitDisplay = () => {
+    return this.props.display.map(tile => {
+      return <p>{tile}</p>
```
Comment (GreyMatteOr) [inline src/Gamepage/Gamepage.js:36]: Make sure to add a `key` prop to these `p` tags or else React will be angry. They need to be unique also T.T

File: src/Gamepage/Gamepage.js [hunk 3]
```diff
@@ -8,30 +8,73 @@ export class Gamepage extends Component {
...
+
+
+  makeGuess = (e) => {
+    e.preventDefault();
+    // console.log("WordSelector -> makeWordToGuess -> this.state.guessWord", this.state.guessWord)
+    this.props.makeGuess(this.state.currentGuess)
+  }
+
+  mapAttempts = () => {
+    return this.props.attempts.map(attempt => {
```
Comment (GreyMatteOr) [inline src/Gamepage/Gamepage.js:29]: Same `key` problem (see below)

**Review by GreyMatteOr — COMMENTED** [review]
Cool beans.

File: src/App.js (hunk 1 above)
Comment (holladayian) [reply src/App.js:19]: This is actually from wreckit ralph

**Comment by Chriscastanuela** [comment]
Looks good to me

File: src/Gamepage/Gamepage.js (hunk 2 above)
Comment (holladayian) [reply src/Gamepage/Gamepage.js:36]: gotcha, I added this as an issue

**Pull request merged**